Update n-series-driver-setup.md

mabicca opened the pull request:
Adding note about nvidia-smi using persistence mode. That should help getting a more quicker output from the command once it's set.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

mabicca committed:
Update n-series-driver-setup.md

Adding note about nvidia-smi using persistence mode. That should help getting a more quicker output from the command once it's set.

GitHubber17 committed:
Update n-series-driver-setup.md
